bigquery/system-test/queries.test.js

```diff
@@ -13,7 +13,7 @@
 
 'use strict';
 
-var example = require('../query');
+var example = require('../queries');
```

Comment:
Not terribly worried about this change (because it needs to be done anyway), but this wasn't _technically_ a `yargs` change. :stuck_out_tongue: